### DISCUSSION TITLE
TextInput icon renders outside of TextInput on first pageload

### DISCUSSION THREAD

**Discussion by Paraprax** [mantinedev/mantine#3884] [Q&A]
Occurs: When any page in the deployed version of my React+TypeScript app is first visited in(at least) Chrome and Safari, on macOS. 

The problem: The icon in my <TextInput/> component renders outside of the form, on a new line, like so - 
![Screenshot 2023-03-23 at 16 08 19](https://user-images.githubusercontent.com/26296581/227341561-5e7a4e9c-f21d-4d54-972e-219a06bd91c4.png)

It remains this way throughout navigating to any page around the site, unless you refresh a page, at which point it renders correctly on all pages for the rest of the session - 
![Screenshot 2023-03-23 at 16 14 02](https://user-images.githubusercontent.com/26296581/227342056-73b10d84-b326-4fd5-951c-b74f8255fdf7.png)

There are no Console errors in DevTools whatsoever during the bug, and no differences in the rendered HTML in the Elements tab between the two visuals. The Icon is an `<Icon>` node from `iconify/react`. Have tried everything I can think of to debug this but no results yet. Any ideas?

**Comment by rtivital** [comment]
What framework do you use?

**Reply by Paraprax** [reply]
Remix

**Reply by rtivital** [reply]
You can find remix issues here – https://github.com/mantinedev/mantine/issues?q=is%3Aissue+label%3Aremix+
Maybe one of the solutions from the community will help you fix the issue
